### PR TITLE
#109 bugfix and regression test

### PR DESCRIPTION
**Pull request by rphh**

### CODE REVIEW

File: sdl2promela/sdl/model.py [hunk 1]
```diff
@@ -1338,10 +1336,13 @@ def __gather_inputs(self):
         for inputSignal in self.source.input_signals:
             input = Input()
             input.name = inputSignal["name"]
-            input.parameters = self.__get_input_parameters(input.name)
-            if len(input.parameters) > 1:
-                raise Exception(f"Too many parameters for {input.name}")
-            self.__assign_input_parameter_type(input, inputSignal)
+            # If signal has property "type" is has a parameter
```
Comment (Lurkerpas) [inline sdl2promela/sdl/model.py:1339]: it has a parameter

File: sdl2promela/sdl/model.py [hunk 2]
```diff
@@ -1338,10 +1336,13 @@ def __gather_inputs(self):
...
+            # parameter name for signals is irrevelant for generation of promela
+            if "type" in inputSignal:
+                param = Parameter("p1")
```
Comment (Lurkerpas) [inline sdl2promela/sdl/model.py:1342]: Why p1?

Comment (rphh) [reply sdl2promela/sdl/model.py:1342]: The translator adds suffix `_input_param` so in the promela it is `p1_input_param`. I have no better idea

Comment (rphh) [reply sdl2promela/sdl/model.py]: Changed to input_param

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**